### ISSUE TITLE
Add Support For Bitcoin EVM

### ISSUE THREAD

**Issue by BitcoinEvm** [AlphaWallet/alpha-wallet-ios#5220]
Add Support For Bitcoin EVM

Chain ID: 2203 

https://explorer.bitcoinevm.com 

https://chainlist.org/chain/2203

Hello Looking To Add Support For Bitcoin Evm Will Be Launching Publicly Within the next 30-60 days and the community would like to use Alpha Wallet As The Native Wallet Thank You.

**Comment by hboon** [comment]
@BitcoinEvm this sounds interesting! I visited https://bitcoinevm.com but I'm not sure. Could you help point to a couple of articles to docs to describe what the chain is about and how it works?

**Comment by BitcoinEvm** [comment]
yes for sure Great Question. This is the documentation page https://docs.bitcoinevm.com

as well as you can see our opensource repos here https://github.com/bitcoinevm 

The Chain Is an EVM Upgrade for  BTC To Add EVM Capabilities To bitcoin. Devs will build on chain dapps for the bitcoin community so bitcoin users can use bitcoin with less fees faster transaction times & have on chain non custodial games & defi apps with their bitcoin.
